File: php/src/Behavioral/Command/DvdCommandNameStarsOn.php
<?

# //DvdCommandNameStarsOn - one of two Concrete Commands

require_once 'CommandAbstract.php';
require_once 'DvdName.php';

class DvdCommandNameStarsOn extends CommandAbstract
{
	private $dvdName = NULL;

	function __construct(DvdName $dvdName)
	{
		$this->dvdName = $dvdName;
	}

	function execute()
	{
		$this->dvdName->setNameStarsOn();
	}
}

File: ruby/src/Behavioral/Command/DvdCommandNameStarsOn.rb
#//DvdCommandNameStarsOn - one of two Concrete Commands

from CommandAbstract import CommandAbstract

class DvdCommandNameStarsOn(CommandAbstract) :

	def initialize(dvdName) :
		self.dvdName = dvdName

	def execute(self) : 
		self.dvdName.setNameStarsOn()
